templates/cli/base/requests/api.twig

```diff
@@ -10,12 +10,16 @@
     }, payload{% if method.type == 'location' %}, 'arraybuffer'{% endif %});
 
 {% if method.type == 'location' %}
```

Comment:
Just FYI, the whitespace slurping twig tags much look better here:
```twig
    {%~ if method.type == 'location' %}
    if (overrideForCli) {
        response = Buffer.from(response);
    }

    fs.writeFileSync(destination, response);
   {%~ endif %}
```

I like to fix them as I touch places in SDK generator